src/ostorlab/scanner/handler.py

```diff
@@ -0,0 +1,229 @@
+"""Module responsible for establishing a connection with NATS."""
+
+import asyncio
+import datetime
+import logging
+import ssl
+import sys
+from typing import Optional
+
+from nats.js import errors
+import nats
+from nats.js import api as js_api
+from nats.js import client as js_client
+
+from ostorlab.scanner.proto.scan._location import startAgentScan_pb2
+
+DEFAULT_PENDING_BYTES_LIMIT = 400 * 1024 * 1024
+
+DEFAULT_CONNECT_TIMEOUT = 20
+
+DEFAULT_MAX_INFLIGHT = 1
+
+DEFAULT_ACK_WAIT = 120
+
+DEFAULT_PUBLISH_ACK_WAIT = 30
+
+logger = logging.getLogger(__name__)
+
+
+class Error(Exception):
```

Comment:
Where is this used. We should inherit `Hanlder` related exception from it if we want to use it anyway.

Reply:
I don't see this used.